src/js/13-announcement-email.js

```diff
@@ -0,0 +1,68 @@
+(function () {
+  'use strict'
+
+  document.addEventListener('DOMContentLoaded', function () {
+    const footer = document.querySelector('.announcement-email')
+    const closeButton = document.querySelector('.close-footer')
+    const submitButton = document.querySelector('.submitButtonEmail')
+    const inputContainer = document.querySelector('.inputContainer')
+    const successfulSentEmail = document.querySelector('.successfulSentEmail')
+    let footerDisplayed = false
+    let footerIsStatic = false
+    let lastScrollTop = 0
+    if (typeof window !== 'undefined') {
+      if (window.localStorage.getItem('footerClosed') === 'true') {
+        footer.classList.add('hidden-footer')
+        return
+      }
+    }
+    window.addEventListener('scroll', function () {
+      const scrollTop = window.scrollY
+
+      if (scrollTop > 50 && !footerDisplayed) {
+        footer.classList.add('show-footer')
```

Comment:
I feel like `footer` isn't very descriptive. Can we use better names like `announcementEmail` (`announcement-email`)